src/app/components/Curation/HierarchicalGrid/index.stories.tsx

```diff
@@ -1,33 +1,24 @@
 import React from 'react';
 
-import { withKnobs, number } from '@storybook/addon-knobs';
-import { ServiceContextProvider } from '../../../contexts/ServiceContext';
-import { withServicesKnob } from '../../../legacy/psammead/psammead-storybook-helpers/src';
-import ThemeProvider from '../../ThemeProvider';
 import HierarchicalGrid from './index';
 import pidginPromos from './fixtures';
-import { StoryProps } from '../../../models/types/storybook';
 
-const Component = ({ service, variant }: StoryProps) => {
+const Component = () => {
   return (
-    <ThemeProvider service={service} variant={variant}>
-      <ServiceContextProvider service={service} variant={variant}>
-        <HierarchicalGrid
-          headingLevel={2}
-          summaries={pidginPromos.slice(
-            0,
-            number('Promo Count', 12, { min: 3, max: 12 }),
-          )}
-        />
-      </ServiceContextProvider>
-    </ThemeProvider>
+    <HierarchicalGrid
+      headingLevel={2}
+      summaries={pidginPromos.slice(
+        0,
+        // number('Promo Count', 12, { min: 3, max: 12 }),
```

Comment:
Did we mean to comment this out, or can we put it back?

Reply:
Good spot, will re-add using `args`